README.md

```diff
@@ -11,16 +11,8 @@
    sam build --use-container
     ```
    This pulls in the required packages and native libraries and creates the deployment packages in .aws-sam/build. It also runs the unit tests. Note the --use-container argument is necessary because the native libraries must be pulled in using a docker image resembling the runtime environment of AWS Lambda
-   
-3. (optional) Run the python unit tests in an environment that mimics the Lambda runtime environment
```

Comment:
Are we completely remove the unit test?

Reply:
No, the unit tests are still there. I'm just removing the instructions regarding the docker way to run them on as the image was deprecated.

To run the unit tests you would need to setup the environment yourself for now. In a future PR we could bring it back.